app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/DbConfiguration.kt

```diff
@@ -30,7 +30,16 @@ data class DbConfiguration(
     /**
      * Time of updating.
      */
-    val cacheTime: ZonedDateTime = ZonedDateTime.now()
+    val cacheTime: ZonedDateTime = ZonedDateTime.now(),
+
+    val exposureConfigurationMinimumRiskScore: Int = 1,                                                    // 1,
```

Comment:
Not helpful prefix in used variable names `exposureConfiguration` please wrap it to a special object or remove the prefix.

Reply:
@falkorichter

Reply:
Agreed to remove prefix